packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -80,6 +80,8 @@ export class TeamsBotV2Impl {
     const bicepContext = {
       plugins: plugins.map((obj) => obj.name),
       configs: bicepConfigs,
+      enableAlwaysOn: isBotNotificationEnabled(),
+      useNode: getLanguage(ctx.projectSetting.programmingLanguage) !== ProgrammingLanguage.Csharp,
```

Comment:
I think there is already a `node` config in `bicepConfigs`. Can we reuse that one?

Reply:
And please also take `updateResourceTemplate` method into consideration when you update configs.

Reply:
done